qtop.py

```diff
@@ -1560,7 +1560,11 @@ def print_y_lines_of_file_starting_from_x(self, file, x, y):
         Justification for implementation:
         http://unix.stackexchange.com/questions/47407/cat-line-x-to-line-y-on-a-huge-file
         """
-        return 'clear;tail -n+%s %s | head -n%s' % (x, file, y)
+        temp_f = tempfile.NamedTemporaryFile(delete=False, suffix='.out', dir=TMPDIR)
+        pre_cat_command = 'clear;(tail -n+%s %s | head -n%s) > %s' % (x, file, y-1, temp_f.name)
```

Comment:
try instead: `(tail -n+%s %s | head -n%s) > %s;clear`; no need to wait that extra time?